### PR TITLE
+ Fix Bot userdata and -cf command parameter

### PR DESCRIPTION
**Pull request by ryuzaki01**
Short Description: 
+ Fix Bot userdata and -cf command parameter
Fixes:
-  Fix -cf argument
- Fix Bot user data parameter

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -112,5 +116,5 @@ web/userdata.js
 data/last-location*.json
 data/catch-ignore.yml
 release_config.json
-web/userdata.js
```
Comment (TheSavior) [inline .gitignore]: Why is this removed?

Comment (TheGoldenXY) [inline .gitignore]: @TheSavior Double decleration.

**Comment by ryuzaki01** [comment]
will fix the conflict..

**Comment by solderzzc** [comment]
Since the review of team, I'd like to close this PR. thanks.

**Comment by TheGoldenXY** [comment]
@solderzzc Why was this PR closed?
